tests/components/modbus/test_init.py

```diff
@@ -63,13 +79,54 @@
 from homeassistant.setup import async_setup_component
 import homeassistant.util.dt as dt_util
 
-from .conftest import TEST_MODBUS_NAME, ReadResult
-
 from tests.common import async_fire_time_changed
 
 TEST_SENSOR_NAME = "testSensor"
 TEST_ENTITY_ID = f"{SENSOR_DOMAIN}.{TEST_SENSOR_NAME}"
 TEST_HOST = "modbusTestHost"
+TEST_MODBUS_NAME = "modbusTest"
+
+
+class ReadResult:
```

Comment:
Why isn't this imported from the library? It represents a read result from the library, right?

This could be a dataclasses.dataclass.

https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass

Reply:
The current pymodbus library have 7-8 different classes as response, something they are cleaning up for 3.0. I decided long time ago not to deal with all the classes which are only slightly different and added again long time ago ReadResult which cover all the pymodbus classes.

Reply:
Removed ReadResult from test_init, and left it where it has always been in conftest.py